### PR TITLE
libvirt 5.8.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
13:39:53 ==> brew install --verbose --build-bottle libvirt
13:43:29 ==> FAILED
13:43:29 Warning: You are using macOS 10.15.
13:43:29 We do not provide support for this pre-release version.
13:43:29 You will encounter build failures with some formulae.
13:43:29 Please create pull requests instead of asking for help on Homebrew's GitHub,
13:43:29 Discourse, Twitter or IRC. You are responsible for resolving any issues you
13:43:29 experience while you are running this pre-release version.

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Somehow the bottles got published, so I had to remove them manually from bintray, and now we can go again. @BrewTestBot test this please

File: Formula/libvirt.rb [hunk 1]
```diff
@@ -41,6 +41,9 @@ def install
 
     args << "ac_cv_path_RPCGEN=#{Formula["rpcgen"].opt_prefix}/bin/rpcgen" if build.head?
 
+    # Work around a gnulib issue with macOS Catalina
```
Comment (chenrui333) [inline Formula/libvirt.rb]: good to know on this!

**Comment by chenrui333** [comment]
> Somehow the bottles got published, so I had to remove them manually from bintray

that is because of me, I did hesitate to pull the trigger for the merge. Thanks for taking care of the Catalina build issue.

**Pull request closed without merging**